resources/qml/TimelineView.qml

```diff
@@ -434,18 +434,32 @@ Page {
 				id: footerContent
 				anchors.left: parent.left
 				anchors.right: parent.right
-
-				Label {
-					id: typingDisplay
-					anchors.left: parent.left
-					anchors.right: parent.right
-					anchors.leftMargin: 10
-					anchors.rightMargin: 10
-
-					color: colors.text
-					text: chat.model ? chat.model.formatTypingUsers(chat.model.typingUsers, colors.window) : ""
-					textFormat: Text.RichText
-				}
+                Rectangle {
+                    id: typingRect
+                    Connections {
+                        target: chat.model
+                        onTypingUsersChanged: {
+                            if (chat.model && chat.model.typingUsers < 1)
+                                typingRect.color = "transparent"
+                            else
+                                typingRect.color = colors.window
+                        }
+                    }
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.leftMargin: 10
+                    anchors.rightMargin: 10
+                    color: "transparent"
+                    height: fontMetrics.height + 10
```

Comment:
I don't think this padding is necessary. I'd probably just make the height of this rectangle depend on the height of the typingDisplay. That way it also accounts for weird oversized unicode characters.